<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QSlider" name="greenASlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Green</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
   </widget>
   <widget class="QSlider" name="redASlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>0</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blueASlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>200</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="rgbThreshCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>RGB Thresholding</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="erosionSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Erosion</string>
    </property>
   </widget>
   <widget class="QSlider" name="dilationSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dilation</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="originalImageRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 1</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="maskedImageRadioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Image 2</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="redBSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="sliderPosition">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="greenBSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blueBSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="redCSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="greenCSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blueCSlider">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveMask</string>
    </property>
   </widget>
   <widget class="RangeSlider" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="blueASlider_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blueBSlider_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="blueCSlider_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="redASlider_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="redBSlider_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="redCSlider_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="greenASlider_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="greenBSlider_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="greenCSlider_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>531</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>150</number>
    </property>
    <property name="value">
     <number>8</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Brightness</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Green-Red</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>81</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Blue-Yellow</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="WebcamCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Webcam</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RangeSlider</class>
   <extends>QWidget</extends>
   <header>RangeSlider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
